tools/labs/Makefile

```diff
@@ -39,6 +39,12 @@ docs:
 	cp -r $(KDIR)/Documentation/output/slides/teaching/_static $(KDIR)/Documentation/output/teaching/
 
 clean::
```

Comment:
Can you please also fix this typo?

Reply:
What typo?

Reply:
clean::

Reply:
That's not a typo. It's Makefile syntax for running multiple clean targets. There's a clean target in this file, and there's another "clean::" in "qemu/Makefile", which is included. By using "clean::" you run both clean targets:
https://web.archive.org/web/20180122002430/http://owen.sj.ca.us/~rk/howto/slides/make/slides/makecolon.html